File: Araba/araba.java
public abstract class araba {
    protected double acceleration;
    protected double velocity;
    
    public araba(double acceleration, double velocity){
        this.acceleration = acceleration;
        this.velocity = velocity;
    }

    public double getAcceleration(){return this.acceleration;}
    public double getVelocity(){return this.velocity;}
    public void setAcceleration(double acceleration){this.acceleration = acceleration;}
    public void setVelocity(double velocity){this.velocity = velocity;}

    public abstract void power();
    public void run(){
        this.velocity += this.acceleration;
    }
}

File: ArabaCsharp/Araba.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace YTM
{
    public abstract class Araba
    {
        protected double acceleration;
        protected double velocity;
        
        public Araba(double acceleration, double velocity){
            this.acceleration = acceleration;
            this.velocity = velocity;
        }

        public double GetAcceleration(){return this.acceleration;}
        public double GetVelocity(){return this.velocity;}
        public void SetAcceleration(double acceleration){this.acceleration = acceleration;}
        public void SetVelocity(double velocity){this.velocity = velocity;}

        public abstract void Power();
        public void Run(){
            this.velocity += this.acceleration;
        }
    }
}